moped-editor/src/components/GridTable/FiltersChips.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { Box, Typography, Chip, Grid } from "@mui/material";
+import makeStyles from "@mui/styles/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  filtersList: {
+    paddingTop: theme.spacing(1),
+    marginRight: "12px",
+    display: "flex",
+    alignItems: "start",
+  },
+  filtersText: {
+    fontFamily: "Roboto",
+    fontSize: ".9rem",
+    color: theme.palette.text.secondary,
```

Comment:
Unrelated to this PR but this is making me realize it is tough to read our secondary text color. We use `#848484` while the MUI default is `#666666` which has better contrast score - [our secondary](https://coolors.co/contrast-checker/666666-ffffff) vs. [MUI default](https://coolors.co/contrast-checker/848484-ffffff).

I'll create an issue that we can refine in the future.